tests/shtest

```diff
@@ -29,50 +29,50 @@ $VALGRIND $Q $JQ -Rne '[inputs] == ["a\u0000b", "c\u0000d", "e"]' $d/input
 ## move all of these into tests/all.test
 
 # String constant folding (addition only)
-n=`$VALGRIND $Q $JQ -n --debug-dump-disasm '"foo"' | wc -l`
-if [ $n -ne 5 ]; then
+nref=`$VALGRIND $Q $JQ -n --debug-dump-disasm '"foo"' | wc -l`
+if [ $nref -gt 8 ]; then
```

Comment:
It's not necessary to compare `$nref` to any particular constant.  It will be more robust to only compare the subsequent counts to `$nref`.